main/xinit/xinitrc

```diff
@@ -45,11 +45,8 @@ if [ -d /etc/X11/xinit/xinitrc.d ] ; then
 	unset f
 fi
 
-# If not present, try the system default
-exec startxfce4
-
-# Fall back to fluxbox
-exec startfluxbox
-
-# Start only terminal as last resort
-exec xfce4-terminal || exec mrxvt || exec urxvt || exec st
+for p in startxfce4 startfluxbox xfce4-terminal \
+		mrxvt urxvt st xterm; do
+	command -v "$p" >/dev/null && exec "$p"
+done
+unset p
```

Comment:
This is never reached unless now suitable program to execute was found. So we could remove this line.

Reply:
Yeah, we can remove the `unset p`.

Reply:
Done